Services/data/devicemodels/chiller-01.json

```diff
@@ -16,11 +16,13 @@
             "pressure_unit": "psig",
             "simulation_state": "normal_pressure"
         },
-        "Script": {
-            "Type": "javascript",
-            "Path": "chiller-01-state.js",
-            "Interval": "00:00:05"
-        }
+        "Interval": "00:00:10",
```

Comment:
why change the structure?  This would allow multiple state scripts that run @ different intervals but I'm not sure we will need this.

Reply:
While the feature would be interesting, we don't have a use case now, and the scheduling implementation is not trivial, so I decided to leave this out for the time being. Note that we're not dropping the feature, because before only 1 script was allowed.